.travis.yml

```diff
@@ -39,7 +39,7 @@ matrix:
     - php: 7.2
       env: STABILITY=dev SYMFONY_VERSION=4.0.*
     - php: 7.1
-      env: STABILITY=dev COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
+      env: STABILITY=dev COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.0 SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
those version come from dev-kit, can you please change that there too. we do not have to merge it from there, just to not introducing same version in a month or so.